lib/parser.js

```diff
@@ -75,6 +75,18 @@ exports.parseParams = function (str) {
   return str.split(/\;\s?/).reduce(paramReducer, {});
 };
 
+exports.parseImageattrParams = function (str) {
+  var list = [];
+
+  str.split(' ').forEach(function (item) {
+    var params = item.substring(1, item.length-1).split(',').reduce(paramReducer, {});
+
+    list.push(params);
+  });
+
+  return list;
```

Comment:
the `forEach` in here should be a `map` so you don't have to think about the `list` temporary.